docs/changelog.md

```diff
@@ -3,6 +3,57 @@
 Maintenance branches are merged before each new release. This change log is
 ordered chronologically, so each release contains all changes described below it.
 
+# v0.23.20 (2023-06-12)
+
+This release fixes a critical regression in Ember HTTP/2. It also upgrades to Scala 3.3.0 LTS.
```

Comment:
Writing this made me a bit queasy. Maybe we should release the fix without forcing the v3.3.0 upgrade, in case anyone is stuck on v3.2.0? But it's a lot of work and I'm not sure how widely Ember H2 is deployed, and the 3.3.0 upgrade seemed to go smoothly.